module/VuFind/src/VuFind/DigitalContent/OverdriveConnector.php

```diff
@@ -1037,34 +1242,50 @@ public function getHolds($refresh = true)
                 if (!empty($response)) {
                     $result->status = true;
                     $result->message = 'hold_place_success_html';
-                    $result->data = $response->holds;
-                    // Check for holds ready for chechout
-                    foreach ($response->holds as $key => $hold) {
-                        if (
-                            !$hold->autoCheckout
-                            && $hold->holdListPosition == 1
-                        ) {
-                            $result->data[$key]->holdReadyForCheckout = true;
-                            // format the expires date.
-                            $holdExpires = new \DateTime($hold->holdExpires);
-                            $result->data[$key]->holdExpires
-                                = $holdExpires->format(
+
+                    if (isset($response->holds)) {
+                        $result->data = [];
+                        $result->data = $response->holds;
+                        // Check for holds ready for chechout
+                        foreach ($response->holds as $key => $hold) {
+                            // check for hold suspension
+                            if (isset($hold->holdSupension)) {
```

Comment:
This if/else could be simplified to:

`$result->data[$key]->holdSupension = $hold->holdSupension ?? false;`